lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_merge_strings_files.rb

```diff
@@ -4,7 +4,8 @@ class IosMergeStringsFilesAction < Action
       def self.run(params)
         UI.message "Merging strings files: #{params[:paths].inspect}"
 
-        duplicates = Fastlane::Helper::Ios::L10nHelper.merge_strings(paths: params[:paths], output_path: params[:destination])
+        destination = params[:destination] || params[:paths].keys.first # Use first path if destination not provided, to do an in-place merge.
```

Comment:
The "Use first path if destination not provided, to do an in-place merge" feature was documented (on the `ConfigItem`) but in practice not implemented.
This missing case was previous missed by the unit tests, because I always provided an explicit value to that param in my tests — even when testing the case for that destination being the same as one of the input paths. This has been fixed by adding [this new test case](https://github.com/wordpress-mobile/release-toolkit/pull/345/files#diff-0b56c04c18a71437c58dce3f37b480545b0b090611e0ba2582319e7760398f46R40-R60).